debian/gbp.conf

```diff
@@ -0,0 +1,10 @@
+[DEFAULT]
```

Comment:
You can delete this file, debian/gbp.conf. This is custom configuration for my use case and has nothing to do with the package build.

Reply:
Sure. Thanks.